app/src/main/java/com/bogdan/codeforceswatcher/features/users/ProfileItemEpoxyModelNew.kt

```diff
@@ -0,0 +1,110 @@
+package com.bogdan.codeforceswatcher.features.users
+
+import android.content.Context
+import android.content.Intent
+import android.view.View
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.bogdan.codeforceswatcher.R
+import com.bogdan.codeforceswatcher.components.compose.theme.AlgoismeTheme
+import com.bogdan.codeforceswatcher.epoxy.BaseEpoxyModel
+import com.bogdan.codeforceswatcher.features.auth.SignInComposeActivity
+import com.bogdan.codeforceswatcher.features.auth.VerificationComposeActivity
+import com.bogdan.codeforceswatcher.features.users.compose.IdentifyView
+import com.bogdan.codeforceswatcher.features.users.compose.ProfileView
+import com.bogdan.codeforceswatcher.features.users.compose.VerifyView
+import io.xorum.codeforceswatcher.features.auth.models.UserAccount
+import io.xorum.codeforceswatcher.features.auth.redux.AuthState
+import io.xorum.codeforceswatcher.redux.analyticsController
+import io.xorum.codeforceswatcher.util.AnalyticsEvents
+import java.text.SimpleDateFormat
+import java.util.*
+
+class ProfileItemEpoxyModel(
+    private val userAccount: UserAccount?,
+    private val authStage: AuthState.Stage,
+
+    ) : BaseEpoxyModel(R.layout.view_compose) {
+
+    init {
+        id("ProfileItemEpoxyModel", userAccount.toString(), authStage.toString())
+    }
+
+    override fun bind(view: View): Unit = with(view) {
+        super.bind(view)
+
+        view.findViewById<ComposeView>(R.id.compose_view).setContent {
+            AlgoismeTheme {
+                ProfileItem(context)
+            }
+        }
+    }
+
+    @Composable
+    private fun ProfileItem(context: Context, modifier: Modifier = Modifier) {
```

Comment:
Should we start breaking lines by arguments when we have 2 arguments at least?

Reply:
Also, composables should ideally be private file-level functions, so they can't directly reference `ProfileItemEpoxyModel` methods and can be previewed. For actions, they should be passed from outside as callbacks.